v2/googlecloud-to-googlecloud/src/main/java/com/google/cloud/teleport/v2/transforms/FileFormatFactorySpannerChangeStreamsToPubSub.java

```diff
@@ -61,6 +62,18 @@ public static WriteToPubSubBuilder newBuilder() {
 
   protected abstract String pubsubTopicName();
 
+  @Nullable
+  protected abstract Boolean includeSpannerSource();
+
```

Comment:
Maybe 'includeSpannerSourceInfo' is a more accurate name here.

Reply:
To me it appears cleaner to keep the function name consistent with the new conig name `includeSpannerSource`. This also aligns with the builder pattern to have same function name with the class member variable.